setup.py

```diff
@@ -11,7 +11,9 @@
     description=DESCRIPTION,
     include_package_data=True,
     dependency_links=open('dependency_links.txt').readlines(),
-    install_requires=open('install_requires.txt').readlines() if sys.version_info.major == 2 else open('install_requires_python3.txt').readlines(),
+    install_requires=open('install_requires.txt'
+                          if sys.version_info.major == 2
+                          else 'install_requires_python3.txt').readlines(),
```

Comment:
This line was failing the flake8 test (is this repo tested via CI?!) so I had to fix it to be able to run the rest of the tests.

Reply:
Our recommended wrapping style would be something like:

```python
    install_requires=open(
        'install_requires.txt' if sys.version_info.major == 2
        else 'install_requires_python3.txt'
    ).readlines(),